recipe/meta.yaml

```diff
@@ -32,8 +32,8 @@ test:
 
 about:
   home: http://www.perl.org/
-  license: Perl Artistic
-  license_family: Other
+  license: Artistic-1.0-Perl
+  license_file: LICENSE
```

Comment:
```suggestion
  license_file:
    - Artistic  # [unix]
    - Copying   # [unix]
    - licenses  # [win]
```

Reply:
You're right, I was just about to ask about how to deal with the windows licenses

Reply:
Yeah it should grab the whole directory in this case. If not, we can dig into it a bit.